### ISSUE TITLE
alt text for images

### ISSUE THREAD

**Issue by erinmgraham** [carpentries-incubator/intro-image-classification-cnn#17]
most images have empty or no alt text - this needs to be remedied

**Comment by erinmgraham** [comment]
added alt text to all figure in all episodes
